spec/views/admin/index.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'admin/index' do
+  let(:ability) { Ability.new(user) }
+  before do
+    allow(controller).to receive(:current_ability).and_return(ability)
+    render
+  end
+  context "when logged in as an admin" do
+    let(:user) { FactoryGirl.build(:admin) }
```

Comment:
This context and user assignment can be removed.  The view doesn't really care if the user is an admin, as the controller has already kept non-admins from seeing it.  I'm not sure about the controller stubbing above, either, that seems unnecessary, but there could be unfamiliar magic at work.